### PR TITLE
Hotfix resolver

### PR DESCRIPTION
**Pull request by volodymyrss**

### CODE REVIEW

File: instruments/astrooda_isgri/astrooda_isgri.inc [hunk 1]
```diff
@@ -256,7 +256,7 @@ function astrooda_isgri($form, &$form_state)
     '#prefix' => '<div class="row">'
   );
 
-  $form['E1_keV']['#default_value'] = 20;
+  $form['E1_keV']['#default_value'] = 25;
```
Comment (burnout87) [inline instruments/astrooda_isgri/astrooda_isgri.inc:259]: This is not needed, and it's actually not the right way to set a default value, just the change that you applied in `.install` file should do the job.

Comment (burnout87) [reply instruments/astrooda_isgri/astrooda_isgri.inc:259]: Also, what is missing is the setting of the `default_value` for `E1_keV` (as well as for `E2_keV`, and I think also `detection_threshold` if I am not wrong) reading it from the `instrument_defaults` var.

**Review by burnout87 — CHANGES_REQUESTED** [review]

File: instruments/astrooda_isgri/astrooda_isgri.inc (hunk 1 above)
Comment (volodymyrss) [inline instruments/astrooda_isgri/astrooda_isgri.inc:259]:
```suggestion
  $form['E1_keV']['#default_value'] = 20;
```

**Review by burnout87 — COMMENTED** [review]
Will you also apply a change within the `.inc` file? And perhaps also for E2_keV and detection_threshold?

File: instruments/astrooda_isgri/astrooda_isgri.install [hunk 2]
```diff
@@ -50,7 +50,7 @@ function astrooda_isgri_install()
     'scw_list' => '005100410010.001,005100420010.001',
     'max_pointings' => 50,
     'integral_data_rights' => 'public',
-    'E1_keV' => 20.0,
+    'E1_keV' => 25.0,
```
Comment (volodymyrss) [inline instruments/astrooda_isgri/astrooda_isgri.install:53]:
```suggestion
    'E1_keV' => 20.0,
```

**Comment by volodymyrss** [comment]
> Will you also apply a change within the `.inc` file? And perhaps also for E2_keV and detection_threshold?

Not here, this is a a hotfix for resolver.

**Review by burnout87 — APPROVED** [review]

**Pull request merged**